test_dump03_dump_restore failing on AWS RDS Aurora

<!-- Please search existing issues to avoid creating duplicates. -->
- EdgeDB Version: master
- OS Version: Ubuntu 20.04

https://github.com/edgedb/edgedb/runs/3977625129?check_suite_focus=true#step:17:4908

```
================================================================================
FAIL: test_dump03_dump_restore (test_dump03.TestDump03)
--------------------------------------------------------------------------------
Py_HashSecret: 09200a45205226a8d0fd76e791efa0088449ec26247f8f0c
random.seed(): 84edbd3232a0edee
--------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/edgedb/edgedb/edb/testbase/server.py", line 128, in wrapper
    self.loop.run_until_complete(
  File "/opt/hostedtoolcache/Python/3.10.0/x64/lib/python3.10/asyncio/base_events.py", line 641, in run_until_complete
    return future.result()
  File "/home/runner/work/edgedb/edgedb/tests/test_dump03.py", line 81, in test_dump03_dump_restore
    await self.check_dump_restore(
  File "/home/runner/work/edgedb/edgedb/edb/testbase/server.py", line 1382, in check_dump_restore
    await check_method(self)
  File "/home/runner/work/edgedb/edgedb/tests/test_dump03.py", line 31, in ensure_schema_data_integrity
    await self._ensure_schema_data_integrity()
  File "/home/runner/work/edgedb/edgedb/tests/test_dump03.py", line 63, in _ensure_schema_data_integrity
    self.assertEqual(result, 2)
AssertionError: 34 != 2
```

--- a/.github/workflows.src/tests-managed-pg.tpl.yml
+++ b/.github/workflows.src/tests-managed-pg.tpl.yml
@@ -351,7 +351,7 @@ jobs:
         EDGEDB_TEST_BACKEND_DSN: postgres://edbtest:${{ secrets.AWS_RDS_PASSWORD }}@${{ needs.setup-aws-aurora.outputs.pghost }}/postgres
       run: |
         edb server --bootstrap-only --backend-dsn=$EDGEDB_TEST_BACKEND_DSN --testmode
-        edb test -j2 -v --backend-dsn=$EDGEDB_TEST_BACKEND_DSN
+        edb test -j1 -v --backend-dsn=$EDGEDB_TEST_BACKEND_DSN
 
   teardown-aws-aurora:
     runs-on: ubuntu-latest

--- a/.github/workflows/tests-managed-pg.yml
+++ b/.github/workflows/tests-managed-pg.yml
@@ -1134,7 +1134,7 @@ jobs:
         EDGEDB_TEST_BACKEND_DSN: postgres://edbtest:${{ secrets.AWS_RDS_PASSWORD }}@${{ needs.setup-aws-aurora.outputs.pghost }}/postgres
       run: |
         edb server --bootstrap-only --backend-dsn=$EDGEDB_TEST_BACKEND_DSN --testmode
-        edb test -j2 -v --backend-dsn=$EDGEDB_TEST_BACKEND_DSN
+        edb test -j1 -v --backend-dsn=$EDGEDB_TEST_BACKEND_DSN
 
   teardown-aws-aurora:
     runs-on: ubuntu-latest

